src/main/java/com/three_tech_solutions/slot_app/controllers/responses/PlanTypesResponse.java

```diff
@@ -0,0 +1,5 @@
+package com.three_tech_solutions.slot_app.controllers.responses;
+
+public record PlanTypesResponse(
```

Comment:
Esto debería ser en singular, ya que el objeto representa a un único tipo de plan


```suggestion
public record PlanTypeResponse(
```